lib/api-helper/listener/executeAutoInspects.ts

```diff
@@ -109,15 +111,12 @@ export function executeAutoInspects(autoInspectRegistrations: Array<AutoInspectR
  *                              ┌──────────┐                      ▼
  *                              │  Error   │                (errors are
  *                              └──────────┘                  ignored)
- */
-
-// tslint:enable
-/**
- * Apply code inspections
+ *
  * @param goalInvocation
  * @param autoInspectRegistrations
  * @param reviewListeners
  */
+// tslint:enable
```

Comment:
This won't show nicely in TypeDoc

<img width="870" alt="screenshot 2018-10-19 at 17 35 59" src="https://user-images.githubusercontent.com/206707/47228467-9795e000-d3c5-11e8-84e5-353b875e1e17.png">